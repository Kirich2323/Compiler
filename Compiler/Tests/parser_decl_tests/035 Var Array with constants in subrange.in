const
    left = 1;
    right = 20;
var
    a : array [left..right] of integer;