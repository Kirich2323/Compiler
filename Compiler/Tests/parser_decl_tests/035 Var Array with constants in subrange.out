main
----------
char                     type 
float                    type 
integer                  type 
left            const integer               1
right           const integer              20
a                         var array [1..20] of integer